frame/support/src/storage/types/counted_map.rs

```diff
@@ -412,6 +412,23 @@ where
 			res
 		})
 	}
+
+	/// Enumerate all elements in the counted map after a specified `starting_raw_key` in no
+	/// particular order.
```

Comment:
is this true?

are they not ordered by key? else if they are in no particular order, why would we have a starting key as useful at all?

Reply:
They are probably ordered by key, however I don't think that we want to "force" ourselves to this "promise", so keeping the api more open to future implementations.

Reply:
Why? You probably mainly want to iterate. 

As the keys are hashed, you also can not be sure that the ordering based on the "decoded" key would be different to the ordering because of hashing the keys.